lib/facter/rh_certificates.rb

```diff
@@ -0,0 +1,36 @@
+def extract_certificates_from_subscription_manager
+  certificates = {}
+  certificate_end_path = '/cert.pem'
+  private_key_end_path = '/key.pem'
+  data = Facter::Core::Execution.exec('subscription-manager config')
+  data = data.gsub("\n", "")
+  data_array = data.scan(/(\S+)\s*=\s* ([^ ]+)/)
+  data_hash = Hash[data_array]
+  consumer_cert_dir = data_hash["consumercertdir"].gsub(/[\[\]]/, "")
```

Comment:
@stbenjam -- will do. though it is strange that I get completely different values
```
ca_cert_dir = [/etc/rhsm/ca/]
consumercertdir = [/etc/pki/consumer]
entitlementcertdir = [/etc/pki/entitlement]
pluginconfdir = [/etc/rhsm/pluginconf.d]
plugindir = [/usr/share/rhsm-plugins]
productcertdir = [/etc/pki/product]
repo_ca_cert = /etc/rhsm/ca/katello-server-ca.pem
```